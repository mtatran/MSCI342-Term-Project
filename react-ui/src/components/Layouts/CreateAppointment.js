import React, { Component, Fragment } from "react";
import CreateAppointmentForm from "../CreateAppointmentForm";

class CreateAppointment extends Component {
  constructor(props) {
    super(props);
    this.state = {
      id: this.props.match.params.id,
      email:this.props.match.params.email
    }
  }
  render() {
    return (
      <Fragment>
        {/* <Header/> */}
        <CreateAppointmentForm user={this.props} id={this.state.id} email={this.props.email}/>
      </Fragment>
    );
  }
}

export default CreateAppointment;

