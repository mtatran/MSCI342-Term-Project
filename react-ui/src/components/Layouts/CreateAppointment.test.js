import React from "react";
import { create } from "react-test-renderer";
import CreateAppointment from './CreateAppointment';
import { BrowserRouter } from "react-router-dom";

describe('Profile Snapshot', () => {
  test('snapshot renders', () => {
    const component = create(<BrowserRouter>
      < CreateAppointment/>
    </BrowserRouter>);
    expect(component.toJSON()).toMatchSnapshot();
  });

});
