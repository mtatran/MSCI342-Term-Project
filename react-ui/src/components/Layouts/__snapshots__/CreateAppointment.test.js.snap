// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Snapshot snapshot renders 1`] = `
<div
  className="BookForm-container"
  style={
    Object {
      "boxSizing": "border-box",
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": 1540,
      "paddingLeft": 15,
      "paddingRight": 15,
      "position": "relative",
    }
  }
>
  <div>
    <p
      className="Title-Style"
    >
      Book Appointment
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "normal",
        "display": "flex",
        "flexGrow": 0,
        "flexShrink": 0,
        "flexWrap": "wrap",
        "justifyContent": "flex-start",
        "marginLeft": -15,
        "marginRight": -15,
      }
    }
  >
    <div
      align="center"
      style={
        Object {
          "boxSizing": "border-box",
          "flexBasis": "100%",
          "flexGrow": 0,
          "flexShrink": 0,
          "left": "auto",
          "marginLeft": "0%",
          "maxWidth": "100%",
          "minHeight": 1,
          "paddingLeft": 15,
          "paddingRight": 15,
          "position": "relative",
          "right": "auto",
          "width": "100%",
        }
      }
    >
      <br />
      <label>
        <textarea
          className="InputReason"
          name="reason"
          onChange={[Function]}
          placeholder="Reason for Booking Appointment"
          type="text"
        />
      </label>
      <br />
      <div>
        (
        300
         Characters left)
      </div>
      <br />
      <form
        onSubmit={[Function]}
      >
        <input
          className="SubmitAppButton"
          type="submit"
          value="Submit!"
        />
      </form>
      <br />
    </div>
  </div>
  <span
    style={
      Object {
        "clear": "both",
        "display": "table",
      }
    }
  />
</div>
`;
